intel_extension_for_transformers/neural_chat/tests/nightly/server/askdoc.yaml

```diff
@@ -30,7 +30,7 @@ retrieval:
     enable: true
     args:
         input_path: "./askdoc"
-        persist_dir: "./out_persist"
+        persist_directory: "./out_persist"
```

Comment:
vector_db_path?

Reply:
yes, we keep the same name in the origin langchain api to avoid extra parameters/variables